src/core_landice/Registry.xml

```diff
@@ -1030,6 +1034,9 @@ is the value of that variable from the *previous* time level!
                 <var name="calvingVelocity" type="real" dimensions="nCells Time" units="m s^{-1}" time_levs="1"
                      description="rate of calving front retreat due to calving, represented as a velocity normal to the calving front (in the x-y plane)."
                 />
+                <var name="calvingVelocityData" type="real" dimensions="nCells Time" units="m s^{-1}" time_levs="1"
```

Comment:
This variable should also be added to the input and restart streams so it can actually be read in.

Reply:
Added